ui/app/components/transit-key-actions.js

```diff
@@ -35,213 +50,142 @@ const TRANSIT_PARAMS = {
   exportKeyType: null,
   exportKeyVersion: null,
   wrappedToken: null,
+  wrappedTTL: '30m',
```

Comment:
This default was set in the `WrapTtl` component. It caused problems to modify on init the passed in arg, so I set the default to "30m", which is what it becomes anyways (set by the `TtlPicker` farther down the line). 

**Note**: the `transit-key-actions/export.hbs` file is the only one I found that uses the `wrapTtl` component. If anyone has more context about the purpose/history of that component, I'd be curious to hear. Maybe it's something we can remove in another PR and replace the use case here with the `TtlPicker` instead?

Reply:
Also, I set the default here because there were errors setting it the old way now that I'm using tracked properties.